cmd/test.go

```diff
@@ -95,7 +95,8 @@ func runTest(cmd *cobra.Command, args []string) error {
 		c := compiler.Config{
 			Logger: logger,
 
-			TestDir: testCmdTestDir,
+			RemoteCluster: cfg.RemoteCluster,
```

Comment:
Not sure if this is correct to use here. I thought that `RemoteCluster` is always false when building locally and true when doing it on Circle. If this is wrong, then I'll introduce a new flag.

Reply:
No this is wrong, we also use it elsewhere.

Reply:
Yes, `RemoteCluster` refers to where the test cluster lives; you can run your test locally (building binaries from your disk) and use a remote cluster for running the tests.